seguro/common/logger.py

```diff
@@ -0,0 +1,41 @@
+"""
+SPDX-FileCopyrightText: 2023 Felix Wege, EONERC-ACS, RWTH Aachen University
+SPDX-License-Identifier: Apache-2.0
+"""
+
+import logging
+from logging.handlers import RotatingFileHandler
+
+
+def init_logger(log_level, logfile, max_bytes=20000, backup_count=5):
+    """Return logger object.
+
+    Arguments:
+        log_level   -- Log level for logfiles
+        logfile     -- Path to logfile
+    """
+    logger = logging.getLogger("brokerClient_logger")
+    logger.setLevel(log_level)
+
+    # Log specified log level to file
+    filehandler = RotatingFileHandler(
```

Comment:
I am not so sure if it makes sense in the SEGuRo context to log to a local file on disk.
How about implementing our own logging handler which writes to S3?

Reply:
E.g. something like this already exists as a PyPi package:
https://github.com/omrikiei/aws_logging_handlers/

Reply:
I implemented a `store_logger` handler for the s3 store.
By default, store and broker clients will try to instantiate this logger and will fall back to a `file_logger` if the store is not reachable (mainly for debugging purposes).